<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>J-Medic Registro de Pacientes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(0, 0, 0);
	font-family: Arial;
	color: rgb(255, 255, 255);
}

QMenuBar{
	text-decoration: underline rgb(255, 255, 255);
}

QLabel{
	font: 75 18pt;
}

QLineEdit,QComboBox,QDoubleSpinBox,QSpinBox{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	font: 75 14pt &quot;Arial&quot;;
}

QPushButton{
  color: #fff !important;
  font-size: 20px;
  font-weight: 500;
  background: #318aac;
  border: 2px solid;
  border-color: #318aac;
  position: relative;
}

QPushButton:before {
  content:&quot;&quot;;
  position: absolute;
  top: 0px;
  left: 0px;
  width: 100%;
  height: 0px;
  background: rgba(255, 255, 255, 0.1);
  transition: all 1s ease;
}

QPushButton:hover:before {
  height: 100%;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; color:#ffffff;&quot;&gt;Registrar Pacientes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Nombre">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido Paterno:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ApellidoP">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Apellido Materno:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ApellidoM">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>  Sexo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_Sexo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Masculino</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Femenino</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>  Peso:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_Peso">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>KG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Estatura:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_Estatura">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mtrs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="input_Edad">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AÃ±os</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Telefono:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Telefono">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>  Tipo de Sangre:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_TipoS">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O-</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Alergias:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Alergias">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_Guardar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagen_Guardar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Guardar/guardar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMostrar_pacientes">
    <property name="title">
     <string>Mostrar pacientes</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuMostrar_pacientes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../image/Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
